hw-10-logging/src/main/java/Demo/Demo.java

```diff
@@ -0,0 +1,12 @@
+package Demo;
```

Comment:
Пакеты в java принято писать в нижнем регистре.a
```suggestion
package demo;
```